### PR TITLE
Various string fixes

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: desktop/onionshare/resources/locale/en.json [hunk 1]
```diff
@@ -157,10 +157,10 @@
     "gui_all_modes_progress_starting": "{0:s}, %p% (calculating)",
     "gui_all_modes_progress_eta": "{0:s}, ETA: {1:s}, %p%",
     "gui_share_mode_no_files": "No Files Sent Yet",
-    "gui_share_mode_autostop_timer_waiting": "Waiting to finish sending",
+    "gui_share_mode_autostop_timer_waiting": "Finishing sending…",
     "gui_website_mode_no_files": "No Website Shared Yet",
     "gui_receive_mode_no_files": "No Files Received Yet",
-    "gui_receive_mode_autostop_timer_waiting": "Waiting to finish receiving",
+    "gui_receive_mode_autostop_timer_waiting": "Finishing reception…",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply desktop/onionshare/resources/locale/en.json:163]: I think it is slightly poorer, but OK.

Comment (comradekingu) [inline desktop/onionshare/resources/locale/en.json:163]:
```suggestion
    "gui_receive_mode_autostop_timer_waiting": "Finishing receiving…",
```

File: desktop/onionshare/resources/locale/en.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
     "gui_settings_authenticate_password_option": "Password",
     "gui_settings_password_label": "Password",
     "gui_settings_tor_bridges": "Connect using a Tor bridge?",
-    "gui_settings_tor_bridges_label": "Bridges help you access the Tor Network in places where Tor is blocked. Depending on where you are, one bridge may work better than another.",
+    "gui_settings_tor_bridges_label": "Bridges help you access the Tor Network in where Tor is blocked. Depending on where you are, one bridge may work better than another.",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply desktop/onionshare/resources/locale/en.json:65]: I think I had some idea here, but it isn't fixed.
Tor is effectively blocked where the user tries to use it, regardless of the cause.
The distinctions are not as clear to users.

Comment (comradekingu) [inline desktop/onionshare/resources/locale/en.json:65]:
```suggestion
    "gui_settings_tor_bridges_label": "Bridges helps your traffic enter the Tor Network where Tor access is blocked. Depending on where you are, one bridge may work better than another.",
```

**Review by micahflee — APPROVED** [review]

**Pull request merged**